sdk/communication/communication-rooms/src/roomsClient.ts

```diff
@@ -320,8 +289,8 @@ export class RoomsClient {
     return tracingClient.withSpan(
       "RoomsClient-AddOrUpdateParticipants",
       options,
-      (updatedOptions) => {
-        this.client.participants.update(roomId, {
+      async (updatedOptions): Promise<void> => {
```

Comment:
This is the bug that customer reported, where we were not waiting for the response from the rest api. Adding an await to wait for the response